tensorboard/webapp/widgets/line_chart_v2/lib/worker/worker_chart_types.ts

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {DataSeriesMetadataMap, Dimension, Extent} from '../internal_types';
+import {RendererType} from '../renderer/renderer_types';
+import {ScaleType} from '../scale_types';
+
+export {RendererType} from '../renderer/renderer_types';
+
+export enum MainToGuestEvent {
+  SERIES_DATA_UPDATE,
+  SERIES_METADATA_CHANGED,
+  SCALE_UPDATE,
+  UPDATE_VIEW_BOX,
```

Comment:
To make sure I understand: `RESIZE` is for when the DOM extent of the
chart container updates, `UPDATE_VIEW_BOX` is for when the chart window
(in data coordinates) updates?